doc/src/fix_indent.rst

```diff
@@ -223,15 +230,12 @@ and 3 means both ends are capped.
 
 The *fflag2* setting allows you to adjust the radius of the rendered
 object for spherical indenters, cylindrical indenters, and planar
-indenters in 2d systems.  In many cases you want to use a value < 0 to
-reduce the radius of the rendered object so that it does not obscure
+indenters in 2d systems.  In many cases you want to use a negative value
```

Comment:
I suggest: A negative value may be necessary to reduce the radius of the rendered object so that it does not obscure nearby atoms.